dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/route/RouteRunApiAuditEntity.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2004-2025, University of Oslo
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its contributors 
+ * may be used to endorse or promote products derived from this software without
+ * specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package org.hisp.dhis.route;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.hisp.dhis.audit.ApiAuditEntity;
+import org.hisp.dhis.audit.AuditAttribute;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+public class RouteRunApiAuditEntity extends ApiAuditEntity {
```

Comment:
I think `Entity` is a particular bad choice for what it is. An entity typically is a domain object of sorts. A better word for an audit is "record" or "entry" highlighting the nature of keeping information for the sake of record keeping.  

I also think `Run` is a bad choice. If I guessing correct what you refer to and want to audit is an invocation of an API endpoint?! A run is suitable for a program or a thread, a task. A method, function or endpoint however is invoked or called, not run. 

I know it is being picky but if names that are slightly off accumulate the understanding of what this is about get more and more diluted leaving the reader in confusion. So we should spend some time finding the right words.